_posts/2021-11-16-release_0_5_x.md

```diff
@@ -104,17 +104,17 @@ as calling
 ```
 
 to switch from executing pipelines locally to running them in the cloud! Check
-out our [functional API guide](https://docs.zenml.io/v/0.5.7/guides/functional-api) to
+out our [functional API guide](https://docs.zenml.io/) to
 learn more about the remaining core concepts or skip straight to
-[the last part](https://docs.zenml.io/v/0.5.7/guides/functional-api/deploy-to-production)
+[the last part](https://docs.zenml.io/user-guide/starter-guide/switch-to-production)
 to see the magic of stacks in action.
 
 ## New post-execution workflow
 
 Inspecting and comparing pipelines after they were executed is an essential part
 of working with machine learning pipelines. That is why we've added a completely
 new
-[post-execution workflow](https://docs.zenml.io/v/0.5.7/guides/post-execution-workflow)
+[post-execution workflow](https://docs.zenml.io/)
```

Comment:
This still has a docs page: https://docs.zenml.io/user-guide/starter-guide/fetch-runs-after-execution

Reply:
Changed it.